src/controllers/TaskDoneTodayController.ts

```diff
@@ -0,0 +1,41 @@
+import { addTaskDoneToday, deleteAllTasksDoneToday, getAllTasksDoneToday } from "@src/api/TasksDoneTodayAPI";
+
+export const completeTask = async (
+  scheduledTaskId: string,
+  goalId: string,
+  scheduledStart: string,
+  scheduledEnd: string,
+) => {
+  try {
+    await addTaskDoneToday({
```

Comment:
you are already catching error in addTaskDoneToday so there's no point of having this try catch